Client/Frontend/Toolbar+URLBar/TabLocationView.swift

```diff
@@ -142,35 +134,26 @@ class TabLocationView: UIView {
         addGestureRecognizer(longPressRecognizer)
         addGestureRecognizer(tapRecognizer)
 
-        let space1px = UIView()
-        space1px.snp.makeConstraints { make in
-            make.width.equalTo(1)
-        }
+        let space1px = UIView.build()
+        space1px.widthAnchor.constraint(equalToConstant: 1).isActive = true
 
         let subviews = [trackingProtectionButton, space1px, urlTextField, readerModeButton, reloadButton]
         contentView = UIStackView(arrangedSubviews: subviews)
         contentView.distribution = .fill
         contentView.alignment = .center
+        contentView.translatesAutoresizingMaskIntoConstraints = false
         addSubview(contentView)
 
-        contentView.snp.makeConstraints { make in
-            make.edges.equalTo(self)
-        }
-
-        trackingProtectionButton.snp.makeConstraints { make in
-            make.width.equalTo(TabLocationViewUX.TPIconSize)
-            make.height.equalTo(TabLocationViewUX.ButtonSize)
-        }
+        contentView.edges(equalTo: self)
```

Comment:
This function was created [here](https://github.com/mozilla-mobile/firefox-ios/blob/fc4eb7dee8077f11a36ee76f180913af0b65c5ab/Client/Extensions/UIViewExtensions.swift#L74)